src/callkit.cs

```diff
@@ -491,13 +493,15 @@ interface CXProvider {
 		CXCallAction [] GetPendingCallActions (Class callActionClass, NSUuid callUuid);
 	}
 
-	[iOS (10, 0)]
+	[iOS (10, 0), Mac (10, 16)]
 	[Introduced (PlatformName.UIKitForMac, 13,0)]
 	[BaseType (typeof (NSObject))]
 	[DisableDefaultCtor]
 	interface CXProviderConfiguration : NSCopying {
 
-		[Export ("localizedName")]
+		[Deprecated (PlatformName.iOS, 14, 0)]
+		[Deprecated (PlatformName.MacOSX, 10, 16)]
```

Comment:
confirm with Apple docs/headers but it might be better to `[NoMac]` this member
IIRC xtro won't complain for missing *and* deprecated API

Reply:
Confirmed by the headers.